packages/scaffold/src/client.ts

```diff
@@ -309,10 +310,17 @@ export class Web3ModalScaffold {
       NetworkController.setAllowUnsupportedChain(options.allowUnsupportedChain)
     }
 
-    if (options.siweControllerClient) {
-      const { SIWEController } = await import('@web3modal/siwe')
+    const { isAppKitAuthEnabled, isAnalyticsEnabled } = await ApiController.fetchProjectConfig()
```

Comment:
What about separating the `isAppKitAuthEnabled` and `isAnalyticsEnabled` into two different functions and make the fetch if it's necessary. 

I see there is a case that code might not go into both if's and in this case this async call will be unnecessary.

Reply:
We always have to fetch `isAppKitAuthEnabled` from now.